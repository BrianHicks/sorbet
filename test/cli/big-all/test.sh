#!/bin/bash
main/sorbet --silence-dev-message --dir test/cli/big-all 2>&1
