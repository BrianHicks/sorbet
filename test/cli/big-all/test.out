big-all.rb:8: Argument does not have asserted type `T.all(Class, IFoo)` https://srb.help/7007
     8 |T.let(Foo.new, T.all(Class, IFoo))
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  Got `Foo` originating from:
    big-all.rb:8:
     8 |T.let(Foo.new, T.all(Class, IFoo))
              ^^^^^^^
  Note:
    `Foo` is not a subtype of `Class`
Errors: 1
